src/models/piranha_arguments.rs

```diff
@@ -268,20 +268,18 @@ impl PiranhaArgumentsBuilder {
 fn get_rule_graph(_arg: &PiranhaArguments) -> RuleGraph {
   // Get the built-in rule -graph for the language
   let piranha_language = _arg.language();
+
   let built_in_rules = RuleGraphBuilder::default()
     .edges(piranha_language.edges().clone().unwrap_or_default().edges)
     .rules(piranha_language.rules().clone().unwrap_or_default().rules)
     .build();
 
+  // TODO: Move to `PiranhaArgumentBuilder`'s _validate
+  // Get the user-defined rule graph (if any) via the Python/Rust API
+  let mut user_defined_rules = _arg.rule_graph().clone();
   // In the scenario when rules/edges are passed as toml files
-  let mut user_defined_rules = if !_arg.path_to_configurations().is_empty() {
```

Comment:
Simplified the code. 

`add_to_seed_rules_group` is also removed now.

Reply:
Might be worth linking from the comment on the source code to a small issue with this context (even just a copy of this comment's text), just in case that `TODO` lingers there for long enough that you forget this plan above 🙂

Reply:
Added issue, updated the todo in the comment 
https://github.com/uber/piranha/commit/d78602292539bbf43a68860008cb8c91c0c5ac6f 
https://github.com/uber/piranha/issues/387